site/docs/v1/tech/account-management/sms-two-factor.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: doc
+title: Account Management - Add SMS Two Factor
+description: Managing Accounts - Add SMS Two Factor
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Add a Two Factor Method - SMS
+
+
+SMS two factor is not enable by default on the `tenant` level. To enable please navigate to `Tenants > icon:edit[type=fas] Edit Tenant > Multi-Factor Tab`
```

Comment:
"not enable"-> "not enabled"

Reply:
I'm not sure what 'on the tenant level' means? I think you mean "SMS MFA is not enabled by default. Configure your tenant by navigating to `[breadcrumb]#Tenants -> Edit Tenant -> Multi-Factor#`."

In general the navigation in docs should use the breadcrumb pattern documented here: https://github.com/FusionAuth/fusionauth-site/blob/master/DocsDevREADME.md